mobify/progressive-web-scaffold
# Update sdk to 0.11.3

## Changes
- Update the SDK to 0.11.3

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Make sure nothing broke as a result of updating to 0.11.3 (see the [changelog here](https://github.com/mobify/progressive-web-sdk/blob/develop/CHANGELOG.md#v0113-february-9-2017)) for example, the fact that Grid was removed as a local component and moved to the SDK.
